### PR TITLE
Decrease cpu load to 30%.

### LINKED ISSUE #58: Issues with the new remapper
Issues with the new remapper I found so far:

- Some Leaves (Oak?) have no collision, you can walk/bug yourself inside of them
- CPU usage is still very high and also spikes now when loading a new chunk
jprofiler_2019-10-10_16-40-39
- Water gets more transparent if you go near it and finally becomes invisible
- Stairs have the collision of a normal full size block, you can only walk downwards. Steps work fine
- Ladders do not work (but probably also do not work in the old remapper
- Wall signs with text on them are invisible. Signs without text work fine
- All blocks have no collision from below when flying/jumping into them
- Pistons are missing the head
- Prismarine lamps (how are they called? the underwater glowstone thingy) are still dirt

### PR DESCRIPTION
**Pull request by EOT3000**
Need review, fixes #58

### CODE REVIEW

**Comment by RednedEpic** [comment]
This doesn't actually resolve the problem, rather it's just a workaround.

**Comment by Tim203** [comment]
^

File: connector/src/main/java/org/geysermc/connector/network/translators/java/world/JavaChunkDataTranslator.java [hunk 1]
```diff
@@ -62,7 +62,11 @@ public void translate(ServerChunkDataPacket packet, GeyserSession session) {
                 levelChunkPacket.setChunkX(packet.getColumn().getX());
                 levelChunkPacket.setChunkZ(packet.getColumn().getZ());
                 levelChunkPacket.setData(payload);
-                session.getUpstream().sendPacket(levelChunkPacket);
+                try {
+                    session.getUpstream().sendPacket(levelChunkPacket);
```
Comment (SupremeMortal) [inline connector/src/main/java/org/geysermc/connector/network/translators/java/world/JavaChunkDataTranslator.java]: This is not how the connection being closed should be handled.

Comment (EOT3000) [reply connector/src/main/java/org/geysermc/connector/network/translators/java/world/JavaChunkDataTranslator.java]: then how should it

**Comment by Malfrador** [comment]
CPU usage is very low (1%) when flying around, but chunks randomly stop loading and in general the chunk loading is very very slow. In general it seems like the view distance is decreased. Chunks up to 4 chunks away from the player load fast, everything farther away does not load at all or loads very slow.
Also some chunks start flickering very rapidly.

**Comment by RednedEpic** [comment]
> CPU usage is very low (1%) when flying around, but chunks randomly stop loading and in general the chunk loading is very very slow. In general it seems like the view distance is decreased. Chunks up to 4 chunks away from the player load fast, everything farther away does not load at all or loads very slow.
> Also some chunks start flickering very rapidly.

It’s because this doesn’t actually solve the problem. @EOT3000 if you come up with a solution, please open a new PR.

**Comment by Tim203** [comment]
The view distance doesn't decrease, because the server sends the chunks and decides how much you can see, we don't change that. Flickering is when the server sends a chunk update, this happens for example when walking far away from a chunk and coming back at it (and call the others unloaded). Those chunks will stay (for whatever reason) in memory until you switch dimension or quit. The issues can probably be fixed but anyway thanks for saying it.

**Pull request closed without merging**